### DISCUSSION TITLE
I assume this wil be changed? maybe old issues removed?

### DISCUSSION THREAD

**Discussion by Retrodad0001** [MonoGame/MonoGame#8146] [General]
![image](https://github.com/MonoGame/MonoGame/assets/9283221/c348d415-ca5f-4097-b06b-ab7b9f509e09)

**Comment by ThomasFOG** [comment]
It will be cleaned, and some of the plans changed. We are still working on the exact details.

We are likely to keep the 3.8.x branch as the future "legacy" branch for Direct 3D 10 and OpenGL support, while moving to 4.x is expected to deprecate those and only support Direct 3D 12 and Vulkan. 3.9 will likely be a transition between the two milestone.
